runelite-client/src/main/java/net/runelite/client/plugins/roguesden/RoguesDenOverlay.java

```diff
@@ -61,34 +70,138 @@ public Dimension render(Graphics2D graphics)
 			return null;
 		}
 
+		graphics.setFont(font);
+
 		LocalPoint playerLocation = client.getLocalPlayer().getLocalLocation();
 
 		plugin.getObstaclesHull().forEach((obstacle, tile) ->
 		{
 			if (tile.getPlane() == client.getPlane() && obstacle.getLocalLocation().distanceTo(playerLocation) < MAX_DISTANCE)
 			{
-				Polygon p = tile.getGameObjects()[0].getConvexHull();
-				if (p != null)
-				{
-					graphics.setColor(Color.CYAN);
-					graphics.drawPolygon(p);
+				Area clickBox = obstacle.getClickbox();
```

Comment:
Make sure to have your opening curly brackets on new lines to make the rest of the code base.